server.js

```diff
@@ -120,10 +120,12 @@ const getNotifications = (ops) => {
         /** Find mentions */
         const pattern = /(@[a-z][-\.a-z\d]+[a-z\d])/gi;
         const content = `${params.title} ${params.body}`;
-        const mentions = _.without(_.uniq(content.match(pattern))
-          .join('@')
-          .toLowerCase()
-          .split('@')
+        const mentions = _.without(
+          _.uniq((content.match(pattern) || [])
```

Comment:
What does this fix? The methods are still in the same order. The only change i see is the extra `... || []`

Reply:
no it isnt the `join` `toLowerCase` and `split` happen before the `_.uniq`

Reply:
And the `|| []` is here in case the match methods return `null`